docs/source/reference/server-api.md

```diff
@@ -15,18 +14,21 @@ This document provides an example of working with the JupyterHub API to
 manage servers for users.
 In particular, we will cover how to:
 
-1. [check status of servers](checking)
-2. [start servers](starting)
-3. [wait for servers to be ready](waiting)
-4. [communicate with servers](communicating)
-5. [stop servers](stopping)
+- [check the status of servers](#checking-server-status)
+- [start servers](#starting-servers)
 
-(checking)=
```

Comment:
Please don't remove anchor targets. If you don't recognize something, it's probably there for a reason and not something that can just be removed.